com.github.vikdevelop.photopea_app.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+rename-desktop-file: photopea.desktop
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea
+    - install -Dm755 -t /app/share/applications photopea.desktop
```

Comment:
you'll need to use `desktop-file-edit` to edit the desktop file whose `icon` property is incorrect. This is why the build failed.

Reply:
> you'll need to use `desktop-file-edit` to edit the desktop file whose `icon` property is incorrect. This is why the build failed.

I created .desktop file for Flatpak in my GitHub repository `phototopea_app` & I created a new release, included it in the manifest, and changed sha256. 
````
[Desktop Entry]
Name=Photopea
Type=Application
Description=Alternative for Adobe Photoshop users
Description[cs]=Alternativa pro uživatele Adobe Photoshopu
Terminal=false
Icon=com.github.vikdevelop.photopea_app
Exec=photopea
Category=Graphics
````